src/babelPluginTsdx.ts

```diff
@@ -88,7 +83,6 @@ export const babelPluginTsdx = babelPlugin.custom(() => ({
         { name: '@babel/plugin-proposal-nullish-coalescing-operator' },
         {
           name: '@babel/plugin-transform-regenerator',
-          async: false,
         },
```

Comment:
Ah, I believe this can be entirely removed as well, because it gets included with `@babel/preset-env` anyway. The only reason it was here was for the override of `async: false`

Reply:
Seems reasonable. Having the option to turn off async transpilation as a user is also better.